cmd/karavictl/cmd/inject.go

```diff
@@ -664,16 +664,21 @@ func (lc *ListChangeForPowerMax) injectIntoReverseProxy(imageAddr, proxyHost str
 	}
 
 	containers := deploy.Spec.Template.Spec.Containers
+	pluginID := deploy.Namespace
 
-	// Remove any existing proxy containers...
+	// Remove any existing proxy containers and check for observability containers...
 	for i, c := range containers {
 		if c.Name == "karavi-authorization-proxy" {
 			containers = append(containers[:i], containers[i+1:]...)
+		} else if c.Name == "karavi-metrics-powerflex" {
+			pluginID = "powerflex"
+		} else if c.Name == "karavi-metrics-powerstore" {
```

Comment:
Do we support powerstore in authorization?

Reply:
We have a story for that in backlog, it is targeted for 1.0